service/src/main/frontend/src/App.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <router-view />
```

Comment:
Probably don't need the vue router since you have a single page/view.

Reply:
The router is very finicky, I think this is required for part of the default extension. If I get rid of the routers it will complain. I think this is just a part of quasar by default.